lnd_multihoplocalchainclaim_test.go

```diff
@@ -23,7 +23,8 @@ func testMultiHopHtlcLocalChainClaim(net *lntest.NetworkHarness, t *harnessTest)
 	// First, we'll create a three hop network: Alice -> Bob -> Carol, with
 	// Carol refusing to actually settle or directly cancel any HTLC's
 	// self.
-	aliceChanPoint, bobChanPoint, carol := createThreeHopHodlNetwork(t, net)
+	aliceChanPoint, bobChanPoint, carol :=
```

Comment:
I think this can just be folded as: 
```go
aliceChanPoint, bobChanPoint, carol := createThreeHopNetwork(
    t, net, true, 
)
```

Same applies to all other instances in this file.

Reply:
Fixed